IDEAS/Buildings/Components/Interfaces/RectangularZoneTemplateInterface.mo

```diff
@@ -426,6 +426,12 @@ partial model RectangularZoneTemplateInterface
   parameter SI.Length PWall = (if hasOutA then lA else 0) + (if hasOutB then lB else 0) + (if hasOutC then lC else 0) + (if hasOutD then lD else 0)
   "Total floor slab perimeter length" annotation(Dialog(tab="Advanced", group="SlabOnGround", enable=(bouTypFlo == IDEAS.Buildings.Components.Interfaces.BoundaryType.SlabOnGround)));
 
+  parameter Boolean hasEmb = false "Set to true if floor is equipped with floor heating or concrete core activation"
+  annotation(Dialog(tab="Floor", group="Floor heating / CCA",
+            enable=(bouTypFlo == IDEAS.Buildings.Components.Interfaces.BoundaryType.InternalWall or
+                    bouTypFlo == IDEAS.Buildings.Components.Interfaces.BoundaryType.BoundaryWall)));
+  parameter Integer nGainEmb = 1 "Number of planes in which CCA or FH pipes are located"
```

Comment:
this should have default (and final?) value of the size of the corresponding port of `intFlo`?

Reply:
I first tried with intFlo.constructionType.nGain but dymola complains that the size is not fixed...

Reply:
how about `conTypFlo.nGai`?